### ISSUE TITLE
signIn does not exist in supabase.auth library

### ISSUE THREAD

**Issue by rscorer** [supabase-community/nuxt3-quickstarter#12]
# Bug report

## Describe the bug

When following the nuxt3 quickstart, it fails when you try to enter an email address and sign in via magic link. An alert shows up 'supabase.auth.signIn is not a function'

## To Reproduce

Follow steps in https://supabase.com/docs/guides/with-nuxt-3

## Expected behavior

When clicking on 'Send Magic Link' I should be emailed a link.

## Additional context

In Auth.vue, replace 

`        const { error } = await supabase.auth.signIn({ email: email.value })`

with 

`        const { error } = await supabase.auth.signInWithOtp({ email: email.value })`

**Comment by rscorer** [comment]
My `package.json` includes `    "@nuxtjs/supabase": "^0.2.1",` whereas this repo includes `"@nuxtjs/supabase": "^0.1.23",` so there must be a discrepancy in the libraries?

**Comment by dshukertjr** [comment]
@rscorer Yup, sorry for the confusion. We actually moved our examples [here](https://github.com/supabase/supabase/tree/master/examples/user-management/nuxt3-user-management), so I will archive this repo and redirect people over there.
